packages/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Link } from 'gatsby'
+import * as css from './Tabs.module.scss'
+import classnames from 'classnames'
+import { TextCaps } from '@elvia/ui'
+import {
+  HddArrowRight,
+  HddArrowLeft,
+} from '@hafslundnett/hdd-icons/hdd-icons-react'
+
+interface LinkItem {
+  name: string
+  url: string
+}
+
+export interface TabsProps {
+  className?: string
+  tabs?: LinkItem[]
+  uri?: string
+}
+
+const useWindowDimensions = () => {
+  const [width, setWidth] = React.useState(window.innerWidth)
+
+  const updateWidth = () => {
+    setWidth(window.innerWidth)
+  }
+
+  useEffect(() => {
+    window.addEventListener('resize', updateWidth)
+    return () => window.removeEventListener('resize', updateWidth)
+  })
+
+  return {
+    width,
+  }
+}
+
+export const Tabs: React.FC<TabsProps> = ({ tabs, uri }) => {
```

Comment:
```suggestion
export const Tabs: React.FC<TabsProps> = ({ tabs = [], uri }) => {
```
Siden tabs er optional kan det være greit å sette en default verdi så slipper man null/undefined sjekker etterpå